From 2beb3e5edd46ea1d87ac87dbb977dce423a1f124 Mon Sep 17 00:00:00 2001
From: Kurt Mahan <kmahan@freescale.com>
Date: Wed, 21 Nov 2007 14:56:19 -0700
Subject: [PATCH] Change elf entry point from _start to _stext.

Changing the symbol was required by CodeWarrior so it could
locate the entry point.

LTIBName: mcfv4e-elf-entry-stext
Signed-off-by: Kurt Mahan <kmahan@freescale.com>
---
 arch/m68k/coldfire/vmlinux-cf.lds |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/arch/m68k/coldfire/vmlinux-cf.lds
+++ b/arch/m68k/coldfire/vmlinux-cf.lds
@@ -4,7 +4,7 @@
 
 OUTPUT_FORMAT("elf32-m68k", "elf32-m68k", "elf32-m68k")
 OUTPUT_ARCH(m68k)
-ENTRY(_start)
+ENTRY(_stext)
 jiffies = jiffies_64 + 4;
 SECTIONS
 {
